glide.yaml

```diff
@@ -65,3 +65,4 @@ import:
   version: v1.1.0
 - package: github.com/onsi/ginkgo
   version: v1.3.1
+- package: github.com/duglin/vlinker
```

Comment:
I agree with @arschles that this dependency should not be glide-managed. Glide is for managing the compile-time dependencies of our own Go code. It's not for managing the installation of tools. We should, instead `go get` this in the `build/build-image/Dockerfile`. Anyone not using containerized build should `go get` themselves.

There's already precedent set for this with `golint` here:

https://github.com/kubernetes-incubator/service-catalog/blob/master/build/build-image/Dockerfile#L29